### PR TITLE
Remove H2 Console related Documentation

### PR DESCRIPTION
**Pull request by ImalshaG**
This PR removes documentation related to H2 console. 

Related issues: https://github.com/wso2/product-is/issues/13163, https://github.com/wso2/product-is/issues/13182

### CODE REVIEW

**Review by nilminiwso2 — APPROVED** [review]

File: en/mkdocs.yml [hunk 1]
```diff
@@ -672,7 +672,6 @@ nav:
             - 'Changing to Remote H2': setup/changing-to-remote-h2.md
             - 'Changing default datasource of BPS': setup/changing-datasource-bpsds.md
             - 'Changing the Default Datasource for Consent Management': setup/changing-datasource-consent-management.md
```
Comment (nilminiwso2) [inline en/mkdocs.yml:674]: Have we updated the `About this Release` page?
Shall we add a section called `Removed features`:
https://is.docs.wso2.com/en/5.12.0/get-started/about-this-release/#deprecated-features

Comment (nilminiwso2) [reply en/mkdocs.yml:674]: @ImalshaG

File: en/docs/get-started/about-this-release.md [hunk 2]
```diff
@@ -42,6 +42,7 @@ WSO2 IS 5.11.0 contains the following new features and enhancements:
 
 - /identity/connect/register API
 - OAuth 1.0
+- H2 Console
```
Comment (nilminiwso2) [inline en/docs/get-started/about-this-release.md:45]: Is this a deprecated feature or a removed feature?

Comment (ImalshaG) [reply en/docs/get-started/about-this-release.md:45]: This feature will be removed from the product. Hence, should we add this under a different heading?

Comment (senthalan) [reply en/docs/get-started/about-this-release.md:47]: Shall we give a small description reasoning out why this has been removed?
- Reported security issue
- Only used in the development time
- No real use case in the production environment

And ask to use standard database client software as the replacement.

**Pull request merged**